packages/react-native-sdk/README.md

```diff
@@ -0,0 +1,59 @@
+# @onflow/react-native-sdk
+
+A React Native library that provides hooks for interacting with the Flow blockchain. It helps you authenticate users, run Cadence scripts and transactions, listen to events, and manage network configuration directly from your components. Fully compatible with Expo.
```

Comment:
Can we add a line here about it being comparable to the React SDK and add a link. We could maybe add a link to the hooks section of the playground as well.

Reply:
Added